### ISSUE TITLE
Allow ally code URLs to include hyphens

### ISSUE THREAD

**Issue by nickshanks** [danpaulson/swgoh.gg.public#465]
The game groups ally codes into three blocks separated by hyphens.
It would be good if the website also allowed this as an optional format for player URLs, e.g. https://swgoh.gg/p/673-178-621/

To be honest I'd prefer this to be the canonical URL, and the existing ones to 301 to this. Even if ally codes were unique to the website, it's both easier to read if you have to type it out manually from a computer screen to a phone screen, and also easier to identify if you have missed a digit. But when the game already groups them into easier chunks, consistency becomes an important factor. UX baby, UX.

**Comment by sanktanglia** [comment]
yeah i def want to provide either option for the urls, not sure if wed swap due to possible regressionms but good idea
